.github/scripts/build-windows.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+choco install -y wasmedge
+choco install -y llvm
```

Comment:
yes needed for wasmedge

Reply:
then it's odd that wasmedge doesn't install llvm as a dependency

Reply:
I will double check but it wouldn't build when I was running locally

Reply:
It is the clang dep, When I have llvm uninstalled and build the wasmedge project locally I get:

```
cargo build -p containerd-shim-wasmedge --no-default-features

thread 'main' panicked at 'Unable to find libclang: "couldn't find any valid shared libraries matching: 
['clang.dll', 'libclang.dll'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can
 be found (invalid: [])"', C:\Users\jstur\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bindgen-
0.65.1\lib.rs:603:31       
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace           
```

Reply:
Ah, ok, that's because of bindgen, which is used in the build script to generate the rust bindings to the c codebase. But I believe its not needed by the final binary (although I might be wrong)
IIRC any clang >= clang 6 is enough for wasmedge bindgen.

Reply:
I've made a note in the script as to why this is needed, maybe can revisit when sorting out https://github.com/WasmEdge/wasmedge-rust-sdk/issues/54